### PR TITLE
[Team Code Review] Springboot Trading Application

### PR DESCRIPTION
**Pull request by hrshrptl**
[Notion Team Ticket](https://www.notion.so/Whale-7ac1183f271a47b3abc1f6676332ba39)
[Personal Scrum Board](https://www.notion.so/b64d009b9a884ce0b80878e71da405ae?v=e82d54657ddf4904aac63ce9de462899)

### CODE REVIEW

File: springboot/src/main/java/ca/jrvs/apps/trading/Application.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// Manually configure DataSource and JdbcTemplate
+@SpringBootApplication(exclude = {JdbcTemplateAutoConfiguration.class,
+    DataSourceAutoConfiguration.class, HibernateJpaAutoConfiguration.class})
+@Component
+public class Application implements CommandLineRunner {
+
+  private Logger logger = LoggerFactory.getLogger(Application.class);
+
+  @Value("$app.init.dailyList")
+  private String[] initDailyList;
```
Comment (jarviscanada) [inline springboot/src/main/java/ca/jrvs/apps/trading/Application.java]: remove

File: springboot/src/main/java/ca/jrvs/apps/trading/TestConfig.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import ca.jrvs.apps.trading.model.config.MarketDataConfig;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.apache.http.conn.HttpClientConnectionManager;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+//@Configuration
```
Comment (jarviscanada) [inline springboot/src/main/java/ca/jrvs/apps/trading/TestConfig.java]: src/test/java/ca/jrvs/apps/trading/TestConfig.java

File: springboot/src/main/java/ca/jrvs/apps/trading/Application.java (hunk 1 above)
Comment (hrshrptl) [reply springboot/src/main/java/ca/jrvs/apps/trading/Application.java]: Resolved in [c011ae9](https://github.com/jarviscanada/jarvis_data_eng_harsh/pull/27/commits/c011ae9a96b2033cfe6e39634e99dd297e8c5f40)

File: springboot/src/main/java/ca/jrvs/apps/trading/TestConfig.java (hunk 2 above)
Comment (hrshrptl) [reply springboot/src/main/java/ca/jrvs/apps/trading/TestConfig.java]: Resolved in [c011ae9](https://github.com/jarviscanada/jarvis_data_eng_harsh/pull/27/commits/c011ae9a96b2033cfe6e39634e99dd297e8c5f40)

**Pull request merged**